src/datePicker/DatePicker.tsx

```diff
@@ -181,7 +182,7 @@ export class DatePicker extends PureComponent<DatePickerProps, DatePickerState>
           months={months}
           firstDayOfWeek={firstDayOfWeek}
           localeUtils={{ ...DayPicker.LocaleUtils, formatMonthTitle: this.formatMonthTitle }}
-          initialMonth={initialDate}
+          initialMonth={initialMonth}
```

Comment:
Is the view scrolled to the selected date?

Reply:
The view is not scrolled

Reply:
The scroll that displays the selected month we be added in the next commit